examples/dbt_dagster_tutorial/dbt_dagster_tutorial/assets.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+import plotly.express as px
+from dagster_dbt import load_assets_from_dbt_project
+
+from dagster import AssetIn, MetadataValue, asset
+from dagster._utils import file_relative_path
+
+
+# These assets would be part of the first stage of the tutorial
+@asset(key_prefix=["jaffle_shop"], group_name="staging")
+def customers_raw() -> pd.DataFrame:
+    data = pd.read_csv("s3://dbt-tutorial-public/jaffle_shop_customers.csv")
```

Comment:
to keep this working until we host the datasets ourselves, i'm leaving this with the s3 version. Once we host the datasets i'll switch it over